File: Modul 2/Python/PRAK205-2310817210014-DamarjatiSuryoLaksono.py
import math
v = input().split(" ")

if len(v) == 1:
    a = float(v[0])
    b = float(input())
    
elif len(v) == 2:
    a, b = map(float, v)

c = pow(b, 2) - pow(a, 2)
alas = math.sqrt(c)
luas = 0.5 * alas * a
keliling = a + b + alas

print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {a:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm ^ 2")

File: Semester-1/Pemrograman-1/Modul-2/C/PRAK205-2310817210014-DamarjatiSuryoLaksono.c
#include <stdio.h>
#include <math.h>

int main() {
    double a, b ;
    scanf("%lf %lf", &a , &b);
    float c = (b*b)-(a*a) ;
    float alas = sqrt (c) ;
    float luas = 0.5*alas*a;
    float keliling = a+b+alas;
    printf("Alas     = %.0f cm \n", alas );
    printf("Tinggi   = %.0f cm \n",a);
    printf("Keliling = %.0f cm \n", keliling );
    printf("Luas     = %.0f cm^2", luas );
return 0;
}
